.pre-commit-config.yaml

```diff
@@ -62,6 +62,21 @@ repos:
             |math|module|note|raw|seealso|toctree|versionadded
             |versionchanged|warning):[^:]
         files: \.(py|pyx|rst)$
+    -   id: non-standard-imports
+        name: Check for non-standard imports
+        language: pygrep
+        entry: >-
+            (from pandas.core.common import|from pandas.core import common
+            |from collections.abc import|from numpy import nan)
```

Comment:
nitpick: id put each of these on separate lines, same on L76-77

Reply:
I'd got the multiline string wrong. `>-` will put a space between each line. Solution is to use `"` and to escape the backslashes (as in the latest commit)